paramiko/kex_gss.py

```diff
@@ -271,7 +281,7 @@ def _parse_kexgss_error(self, m):
         maj_status = m.get_int()
         min_status = m.get_int()
         err_msg = m.get_string()
-        lang_tag = m.get_string()   # we don't care about the language!
+        m.get_string()   # we don't care about the language!
```

Comment:
interesting that unused m string was handled differently here (even though it also immediately raises an exception)